CRM/Member/BAO/Query.php

```diff
@@ -312,6 +312,18 @@ public static function whereClauseSingle(&$values, &$query) {
         }
         $query->_tables['civicrm_membership'] = $query->_whereTables['civicrm_membership'] = 1;
         return;
+
+      case 'member_override':
```

Comment:
Yeah - I'm not a fan of all the long switch statements. IDEALLY by the time the query gets past 'convertFormValues' the syntax should follow a consistent format - which for this one it does. 

I know Monish did some stuff to standardise qill handling too - but this might need some 'person written words' rather than calculated words.